init.sh

```diff
@@ -1,63 +1,86 @@
 #!/bin/bash
 
+VERSION="$1"
+VERSION="${VERSION,,}"
+if [[ ${VERSION} != "pro" ]] && [[ ${VERSION} != "oss" ]]
+then
+  echo "Usage:"
+  echo "  ./init.sh pro"
+  echo "  ./init.sh oss"
+  exit 1
+fi
+
 set -e
 
 SCRIPT_BASE_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 # Login Information
 OSE_CLI_USER="admin"
 OSE_CLI_PASSWORD="admin"
-OSE_CI_PROJECT="ci"
+OSE_NEXUS_PROJECT="nexus"
 
 # Login to OSE
 echo
 echo "Logging into OSE..."
+echo "=================================="
 echo
-oc login -u ${OSE_CLI_USER} -p ${OSE_CLI_PASSWORD} >/dev/null 2>&1
+oc login -u ${OSE_CLI_USER} -p ${OSE_CLI_PASSWORD}
 
 # Create CI Project
 echo
-echo "Creating new CI Project (${OSE_CI_PROJECT})..."
+echo "Creating new Nexus Project (${OSE_NEXUS_PROJECT})..."
+echo "=================================="
 echo
-oc new-project ${OSE_CI_PROJECT} >/dev/null 2>&1
+oc new-project ${OSE_NEXUS_PROJECT}
 
 echo
 echo "Configuring project permissions..."
+echo "=================================="
 echo
-
 # Grant Default CI Account Edit Access to All Projects and OpenShift Project
-oc policy add-role-to-user edit system:serviceaccount:${OSE_CI_PROJECT}:default -n ${OSE_CI_PROJECT}
+oc policy add-role-to-user edit system:serviceaccount:${OSE_NEXUS_PROJECT}:default -n ${OSE_NEXUS_PROJECT}
 
-# Process Nexus Template
-echo
-echo "Processing Nexus OSS Template..."
-echo
-oc create -f "${SCRIPT_BASE_DIR}/nexus-oss.json" -n ${OSE_CI_PROJECT} >/dev/null 2>&1
+if [[ ${VERSION} == "oss" ]]
```

Comment:
> Seems you can only have one installation per project as they both attempt to create an rhel7 image stream.

Maybe we need to separate the creation of the rhel7 image stream from the creation of the nexus oss/pro templates?